Exiled.API/Features/Roles/Scp106Role.cs

```diff
@@ -78,6 +78,6 @@ public float CaptureCooldown
         /// Contains SCP-106.
         /// </summary>
         /// <param name="container">The player who recontained SCP-106.</param>
-        public void Contain(Player container) => Script.Contain(container.Footprint);
+        public void Contain(Player container) => Script.Contain(container?.Footprint ?? Server.Host.Footprint);
```

Comment:
Instead of giving it the servers hosts footprint, add a proper null check to avoid code execution + avoid weird issues

Reply:
If it's null, throw an exception or something?